src/routes/game/components/dialogs/components/FiveDialog.vue

```diff
@@ -47,14 +47,20 @@ const { player } = storeToRefs(gameStore);
 const selectedCard = ref();
 const emit = defineEmits(['resolve-five']);
 
-const disableButton = computed(() => !player.value.hand.length || !selectCard.value);
+const disableButton = computed(() => player.value.hand.length && !selectedCard.value);
```

Comment:
The computed prop for disabling the button was bugged so that it wasn't disabled when you'd failed to select a card. I also clarified its logic here by removing negations. 

The button should be disabled if you have cards in your hand but haven't selected one for discard